### DISCUSSION TITLE
Is there a way to use Google Scholar to create a dataset?

### DISCUSSION THREAD

**Discussion by SagevdBrand** [asreview/asreview#741] [Q&A] [answered]
More specifically, is it possible to export the search results of Google Scholar?

This question has come up a few times before during workshops. Therefore I want to invite the community to see if we can come up with answers, maybe even tools to facilitate this! 💪

**Comment by boer0107-zz** [comment, marked as answer]
Yes it is possible, but with two important limitations:
-You can only download the first 1.000 results from Google Scholar
-Google Scholar does not offer abstracts, but only exports pieces of text from the publication (one or two lines). You should preprocess the dataset to add abstracts (https://github.com/asreview/asreview/discussions/553).

[Publish or Perish](https://harzing.com/resources/publish-or-perish) is a tool designed to calculate your H-index using Google Scholar, but it can also be used to perform a search on Google Scholar and download the results, limited to 1.000 publications..

You can also download results in smaller batches (20 results) into Zotero using the [Zotero Connector](https://www.zotero.org/download/connectors).
